node/manifest.ts

```diff
@@ -43,6 +43,17 @@ interface GenerateManifestOptions {
   layers?: Layer[]
 }
 
+const removeEmptyConfigValues = (functionConfig: EdgeFunctionConfig) =>
+  Object.entries(functionConfig).reduce((acc, [key, value]) => {
+    if (value && !(Array.isArray(value) && value.length === 0)) {
+      return { ...acc, [key]: value }
+    }
+    return acc
+  }, {} as EdgeFunctionConfig)
+
+const hasAnyConfigValues = (functionConfig: EdgeFunctionConfig) =>
```

Comment:
I don't love the fact that we're defining above what a config object is and then having another allow-list here, so adding a new property will mean adding it in two places. I can see people getting tripped up by that.

I don't have an obvious alternative to offer without getting into more complex schema validation, so I'm happy to defer to you.

Reply:
I tried it this way because [on this line](https://github.com/netlify/edge-bundler/pull/348/files#diff-e659ded745234fd55b239ea6728eef800523583941fd1542d88b44373cd39e8aL56) we were checking some of the values. 

What I could do, is allow for anything in the config to be passed to the function_config. This would mean that we get anything defined in it. Including `cache: manual` for example. Then we only need to filter out any config that is empty.